wsgiadapter.py

```diff
@@ -92,17 +92,22 @@ def send(self, request, *args, **kwargs):
 
         urlinfo = urlparse(request.url)
 
+        content_type = None
         if not request.body:
             data = b''
         # requests>=2.11.0 makes request body a bytes object which no longer needs
         # encoding
         elif isinstance(request.body, bytes):
             data = request.body
+            # TODO: remove?
+            content_type = 'application/octet-stream'
         else:
             data = request.body.encode('utf-8')
+            # TODO: remove?
+            content_type = 'text/plain'
 
         environ = {
-            'CONTENT_TYPE': request.headers.get('Content-Type', 'text/plain'),
+            'CONTENT_TYPE': request.headers.get('Content-Type', content_type),
```

Comment:
Thoughts on defaulting to `""` (empty string). I'm also leaning towards not automatically setting it based on the body if it's not in the request headers.